README.md

```diff
@@ -87,24 +88,24 @@ ebcli tx ethbridge create-claim --help
 
 # Now you can test out the ethbridge module by submitting a claim for an ethereum prophecy
 # Create a bridge lock claim (Ethereum prophecies are stored on the blockchain with an identifier created by concatenating the nonce and sender address)
-ebcli tx ethbridge create-claim 0 0x7B95B6EC7EbD73572298cEf32Bb54FA408207359 $(ebcli keys show testuser -a) $(ebcli keys show validator -a --bech val) 3eth lock --from=validator --chain-id=peggy --yes
+ebcli tx ethbridge create-claim 0 0x7B95B6EC7EbD73572298cEf32Bb54FA408207359 $(ebcli keys show testuser -a) $(ebcli keys show validator -a --bech val) 3stake lock --from=validator --chain-id=peggy --yes
```

Comment:
why is this being changed to stake?

Reply:
Changed back to stake in https://github.com/cosmos/peggy/pull/86/commits/ea3743ad6b1a4f8031e72b636d4a2e5a6a5eab58. Was originally changed while working on an issue I encountered - my testuser account had a balance of 27 eth and ~50 stake, but when entering 'eth' burn/lock weren't being recognized as valid txs and weren't decrementing the eth balance, yet stake was working. Must have been an old state saved on my Bridge, because once I minted more eth tokens to testuser burn/lock began working again.